packages/ag-charts-enterprise/src/features/annotations/line/lineScene.ts

```diff
@@ -29,7 +29,7 @@ export class LineScene extends LinearScene<LineTypeProperties> {
     private endCap?: CapScene;
 
     constructor() {
-        super();
+        super({ layer: true });
```

Comment:
Not ideal, but needs to be a different canvas to prevent it from erasing other drawings on the same canvas.